### DISCUSSION TITLE
articles/enlightenment-now

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#174] [Comments]
# articles/enlightenment-now

谈到「启蒙」一词，很多人会想到18世纪发生在欧洲的「启蒙运动」，这是一场哲学和文化的运动，也是「文艺复兴运动」后人类再一次的思想解放。如果说「文艺复兴运动」使人们从宗教的思想禁锢中解放出来，那么「启蒙运动」则使人们利用理性来认知世界。用康德的话说：“启蒙运动就是人类脱离自己所加之于自己的不成熟状态，而不成熟的状态就是不经别人的引导，就对运用自己的理智无能为力。”

https://hutusi.com/articles/enlightenment-now

**Comment by huangwb8** [comment]
感谢分享(～￣▽￣)～

**Reply by hutusi** [reply]
好书共读😀

**Comment by liugezhou** [comment]
真漂亮

**Reply by hutusi** [reply]
书吗？还是插图？

**Reply by liugezhou** [reply]
你写的文章   Cool！

**Reply by hutusi** [reply]
谢谢你的鼓励：）

**Comment by exbob** [comment]
可太需要了

**Reply by hutusi** [reply]
yes!

**Comment by yihui** [comment]
这十三个方面我基本上都认同，唯有“十二、生活质量”，要是与你的[《新穷人》读书笔记](https://hutusi.com/articles/the-new-poor)对比，感觉有些矛盾，而我是更倾向于认同后者的。

**Reply by hutusi** [reply]
确实。《瓜亚基印第安人编年史》中写道：印第安人每天搜寻食物的时间不超过三小时，其他二十一小时都被用在闲暇乐事上，睡觉、庆祝、找乐子、讲故事。相对于此，当今工薪阶层的人们可真是社畜呀。不过相对于农业社会和工业社会早期的大部分人来说，当代人生活质量可能确实要高些，至少，大部分人不用为生计和果腹而奔波，只是被无穷的物质欲望卷的厉害。

BTW, 很开心收到yihui的评论。

**Reply by yihui** [reply]
是啊，从物质角度说，生活质量肯定是提高了无数倍。只是这个质量的提高带来了精神上的副作用，就是好像我们应该无止境提高。历史上人类的生活是一场有限游戏（三小时找完食物吃饱就结束），现在的生活变成了一场无限游戏、一场终点线一直往前挪动的马拉松；对国家而言，经济必须保持增长（从而消费主义停不下来）；对公司来说，用户数量必须保持增长；对个人来说，财富、技能都必须保持增长；很少有人愿意喊一声“够啦！”然后主动停下来。

你的文章我断断续续看了一些天了，写得很棒，待我有空时还会继续发评论。

**Comment by NOeSIS-yu** [comment]
就好像我现在不停阅读博客一样，停不下来，因为脑袋空空，哈哈哈

**Reply by hutusi** [reply]
”空“的状态好呀，最近我正在读《老子》，这就是老子所推崇的”不如守中“的中空的状态~~填满东西就僵化了~~~

**Comment by weijinglin** [comment]
写得真好！可以问问作者平时怎么阅读的吗？感觉没有系统的阅读方法恐怕很难在阅读完一本书籍后有如此系统的见解了

**Reply by hutusi** [reply]
谢谢。我现在主要在微信读书上阅读，直接在上面做笔记，然后会导出到notion上进行整理。

**Comment by syscn** [comment]
除非将现有的整个学校教育系统推翻，否则现在的人要启蒙，得靠社会来教了。
